MovieSharing/VideoAPIService.swift

```diff
@@ -0,0 +1,57 @@
+//
+//  VideoRequest.swift
+//  MovieSharing
+//
+//  Created by Mariam Korashi on 29.09.21.
+//
+
+import Foundation
+
+
+
+struct VideoAPIService {
```

Comment:
Ok, so, I think an iterative approach for adjusting the architecture works best now. So here would be the first tasks for adjustment:
1. As you will need several API requests also for things that aren't just video lists, I would suggest to not have a specific "VideoAPIService" but a more general class that handles API requests. In DENT we have the convention to call theses classes Managers, so "APIManager" would be a good name. So create a manager that just handles the request.
2. The APIManager should get all the relevant information for the request from another Manager that is a bit more specific to what kind of information you want to receive and puts together the information and also handles the result from the APIManager. Create that Manager for your videos.
3. Make both managers classes and singletons.

Just implement this how you think it works best and we can then go over your approach to this and add more details to adjust the Networking layer :)